frontside/lib/views/home/home.dart

```diff
@@ -9,17 +72,23 @@ class Home extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(
-        title: Text("Home"),
-        centerTitle: true,
-      ),
-      body: Container(
-        width: MediaQuery.of(context).size.width,
-        child: Image(
-          image: AssetImage('assets/example_screens/mapsicle-map.png'),
-          fit: BoxFit.none,
+      body: Column(children: [
+        Padding(
+          padding: EdgeInsets.all(16),
+          child: TextField(
+            readOnly: true,
```

Comment:
why this is read only textfield?

Reply:
you don't write anything in this input field, it only activates the `showSearch` function after tap

I believe this is going to change after the design arrives anyway